src/crypto-browser.js

```diff
@@ -34,9 +35,10 @@ function webCryptoHash (type) {
       return
     }
 
-    return res.then((arrbuf) => {
-      callback(null, new Buffer(new Uint8Array(arrbuf)))
-    }).catch((err) => callback(err))
+    nodeify(
+      res.then((raw) => new Buffer(new Uint8Array(raw))),
+      callback
+    )
```

Comment:
It is different though, examples in the npm package show a complete wrap of the Promise, here we are waiting the promise to resolve to convert it to a buffer and then calling the callback through nodeify.

It is missing the `res.catch(err)`